### PR TITLE
re-doing db using mysql, adding react

### PR DESCRIPTION
**Pull request by michchen**

### CODE REVIEW

File: database/db.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  database: "amazon"
+});
+
+con.connect(err => {
+  if (err) {
+    console.log("error");
+  }
+});
+
+const getAllProducts = function(cb) {
```
Comment (mattminwoolee) [inline database/db.js]: Will you ever need to retrieve all products?

File: server.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+const db = require("./database/db.js");
+
+app.use("/", express.static(__dirname + "/client"));
+
+app.get("/get", (req, res) => {
+  if (req.query.id === undefined) {
```
Comment (mattminwoolee) [inline server.js]: What is req.query.id?

File: client/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+<style type="text/css">
+
+</style>
+</head>
+<body>
+index.html body
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (avincenthill) [inline client/index.html]: Recommend npm install jquery

File: database/db.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+const getProduct = function(id, cb) {
+  con.query(`SELECT * FROM products WHERE ID=${id}`, function(err, res) {
+    cb(res);
+  });
+};
+
+module.exports.getAllProducts = getAllProducts;
+module.exports.getProduct = getProduct;
```
Comment (avincenthill) [inline database/db.js]: Recommend structuring these export methods into one object

File: database/db.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+const mysql = require("mysql");
+
+var con = mysql.createConnection({
+  url: "localhost",
+  user: "root",
+  password: "",
+  database: "amazon"
+});
+
+con.connect(err => {
```
Comment (nr431207) [inline database/db.js]: I recommend adding a console.log for when the con.connect is successful.

File: package.json [hunk 6]
```diff
@@ -1,17 +1,26 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/8-bit-beauty/component-michchen/issues"
   },
+  "homepage": "https://github.com/8-bit-beauty/component-michchen#readme",
+  "dependencies": {},
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
```
Comment (avincenthill) [inline package.json]: I think express is not a dev-only thing... not sure tho

File: server.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+      res.send(data);
+    });
+  } else {
+    db.getProduct(req.query.id, data => {
+      res.send(data);
+    });
+  }
+});
+
+app.listen("9000", () => {
```
Comment (avincenthill) [inline server.js]: Use process.env.PORT environmental variable for port for easier deployment and integration, lookup dotenv npm package

**Pull request closed without merging**